src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,80 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+public class HomePage extends BasePage {
+    public HomePage(WebDriver driver, WebDriverWait wait, Actions actions) {
+        super(driver, wait, actions);
+    }
+
+    By firstScroll = By.cssSelector(".playlist:nth-child(3)");
+    By spellTarget = By.cssSelector("[name='name']");
+    By getRenamedScrollAch = By.cssSelector("div.success.show");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```